Lib/fontTools/feaLib/parser.py

```diff
@@ -838,6 +839,35 @@ def parse_table_BASE_(self, table):
             elif self.cur_token_ == ";":
                 continue
 
+    def parse_table_OS_2_(self, table):
+        statements = table.statements
+        numbers = ("FSType", "TypoAscender", "TypoDescender", "TypoLineGap",
+                   "winAscent", "winDescent", "XHeight", "CapHeight",
+                   "WeightClass", "WidthClass", "LowerOpSize", "UpperOpSize")
+        ranges = ("UnicodeRange", "CodePageRange")
+        os2 = {}
+        while self.next_token_ != "}":
```

Comment:
I don’t know, but that is not the only place that does this. So I think we need to check this and if it is actually an issue fix all of them at once.